src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1261,6 +1270,49 @@ def _get_log(url, user_name, password, log_file=None):
     r.release_conn()
 
 
+def _print_site_logs(api_string, number_of_lines, byte_limit, http, headers):
+    import sys
+    href_count = api_string.count('https')
+
+    std_encoding = sys.stdout.encoding
+    if href_count > 1:
+        logger.info('Will show last %i lines of logs per instance', number_of_lines)
+
+    for x in range(0, href_count):
+        index_of_path = api_string.index('\",\"path')
```

Comment:
From my experience of using KUDU, the service does support JSON response payload, can you double check? if yes, the parsing logics here, which is very fragile, can be handled by `json.loads`. So please try best to avoid the string macth here